source/ucumLhcUtils.js

```diff
@@ -183,29 +178,29 @@ export class UcumLhcUtils {
 
     // if neither unit is a molar/eq unit, return 'normal'
     if (!(fromUnit.isMolarUnit() || toUnit.isMolarUnit())) {
-      return conversionType;
+      return 'normal';
     }
      
-    // handle eq <-> eq conversions
+    // handle eq <-> eq conversions as normal
     if (fromUnit.isEquivalentUnit() && toUnit.isEquivalentUnit()) {
-      conversionType = 'eq<->eq';
+      return 'normal';
     }
     // handle eq <-> mol/mass conversions
     else if (fromUnit.isEquivalentUnit()) {
-      conversionType = toUnit.isMolarUnit() ? 'eq->mol' : 'eq->mass';
+      return toUnit.isMolarUnit() ? 'eq->mol' : 'eq->mass';
     } else if (toUnit.isEquivalentUnit()) {
-      conversionType = fromUnit.isMolarUnit() ? 'mol->eq' : 'mass->eq';
+      return fromUnit.isMolarUnit() ? 'mol->eq' : 'mass->eq';
     }
     // handle mol <-> mass conversions
     else if (fromUnit.isMolarUnit() && toUnit.isMolarUnit()) {
-      conversionType = 'mol<->mol';
+      return 'normal';
     } else if (fromUnit.isMolarUnit()) {
-      conversionType = 'mol->mass';
+      return 'mol->mass';
     } else if (toUnit.isMolarUnit()) {
-      conversionType = 'mass->mol';
+      return 'mass->mol';
     }
 
-    return conversionType;
+    return 'normal';
```

Comment:
Is this line ever executed?  In general I think it better not to have multiple returns, though there are exceptions.

Reply:
I initially was using the early return pattern for specific cases which were 'normal', and thought it may be better to update the rest of the function to be consistant.